core/src/main/res/layout/list_item_search.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Kiwix Android
+  ~ Copyright (c) 2020 Kiwix <android.kiwix.org>
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program. If not, see <http://www.gnu.org/licenses/>.
+  ~
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content">
+
+  <CheckedTextView
```

Comment:
the text can flow underneath the new tab button
![image](https://user-images.githubusercontent.com/3358282/86092356-4a787580-baa5-11ea-9e03-dd570f44a34d.png)

This can even be previewed in the preview of AS by using `tools` attributes

![image](https://user-images.githubusercontent.com/3358282/86092638-b35fed80-baa5-11ea-9ed6-f72bbddd836e.png)

This is solvable using constraints.

We also don't have to use CheckedTextView as we don't want to check this, a simple TextView will suffice.

Additionally the new tab button has no feedback when pressed like the item row, an ImageButton might help here or perhaps just an ImageView with `style="?borderlessButtonStyle"`